drivers/serial/Kconfig.uart_sam

```diff
@@ -49,19 +32,6 @@ config UART_SAM_PORT_1
 
 if UART_SAM_PORT_1
 
-config UART_SAM_PORT_1_NAME
-	string "UART1 Device Name"
-	default "UART_1"
-	help
-	  Device name allows user to obtain a handle to the device object
-	  required by all driver API functions. Device name has to be unique.
-
-config UART_SAM_PORT_1_BAUD_RATE
-	int "UART1 Baud Rate"
-	default 115200
-	help
-	  UART1 baud rate to be set at boot.
-
 choice UART_SAME70_PORT_1_PIN_TX
```

Comment:
Yeah, pin ctrl is a bit more involved, @erwango is working on this for STM so once we see that we can start looking at doing that across the various SoCs.